mate-base/mate-applets/mate-applets-1.27.1.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
Looks like this bump will fix https://bugs.gentoo.org/919089, so tag that (mentioned in release notes)